src/main/java/org/mskcc/oncokb/curation/web/rest/VariantRecommendController.java

```diff
@@ -44,18 +45,17 @@ public ResponseEntity<List<JSONObject>> requestData(@PathVariable String filenam
                 throw new BadRequestAlertException("File is empty", ENTITY_NAME, "fileempty");
             }
             String[] headers = headerLine.split("\t");
-            List<JSONObject> jsonList = new ArrayList<>();
+            JSONArray jsonArray = new JSONArray();
             String line;
             while ((line = reader.readLine()) != null) {
                 String[] data = line.split("\t");
                 JSONObject jsonObject = new JSONObject();
                 for (int i = 0; i < headers.length; i++) {
                     jsonObject.put(headers[i], "n/a".equals(data[i]) ? null : data[i]);
                 }
-                jsonList.add(jsonObject);
+                jsonArray.put(jsonObject);
             }
-
-            return ResponseEntity.ok(jsonList);
+            return ResponseEntity.ok(jsonArray.toString());
```

Comment:
Could you double that the content type is `Content-Type: application/json;` in the response header?

Reply:
Done